### DISCUSSION TITLE
Use music folder outside of the Docker container

### DISCUSSION THREAD

**Discussion by virse** [AzuraCast/AzuraCast#5231] [Q&A] [answered]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Stable Channel

### Current AzuraCast Version

0.15.2 Stable

### What happened?

Hey there awesome work with AzuraCast!

I would like to mount my music collection outside of the docker container to be available within the dockerized Azuracast.

The collection is on my server on _/music/MP3-Database/_

What do I have to change that this folder is available within the Docker installation in my sender? I already tried this: https://github.com/AzuraCast/AzuraCast/issues/3131

But it seems to be an outdated version the solution refers to?

Where do I have to put _/music/MP3-Database:/var/azuracast/stations/cartoonland/media/_ (when this the right syntax for the extra-volume)? In _/var/azuracast/docker-compose.yml_ which will be overwritten I assume when updating the stack? Or anywhere else? Do I have to reinstall with the docker.sh? Or is it enough to make the changes and rebuild the containers?

Any help on this? I read all the possible issues and teh documentation.

Thank you very much.

Mike

**Comment by Vaalyn** [comment]
Please follow [this tutorial](https://docs.azuracast.com/en/administration/docker#mounting-a-directory-into-a-station) in our documentation for mounting a directory into a station.

**Comment by mcc-destroyer** [comment, marked as answer]
Hi, if you go to the installation folder, create a file called docker-compose.override.yml.
Inside that file, copy the following:


services:
  web:
    volumes:
      - /path/on/host/computer:/var/azuracast/stations/<STATION_NAME>/media/

change the path on host computer to where your music files are, and change <STATION_NAME> to your stations name thats in the admin/stations panel.

Save the file and stand up your docker container.
 You might rin into trouble with URI errors or errors that say Nonexistent file. 
I'm in the same boat there, and havent found a way to get my lists to play due to some error.

Hope I've been of help.
